### ISSUE TITLE
Python output without encoding it?

### ISSUE THREAD

**Issue by analyserdmz** [n1nj4sec/pupy#891]
I'm trying to debug a "invalid syntax error" against **py** and **pyinst** files but I am unable to produce a python file that's not encoded. I've created the payload by using the following command.

`sudo ./pupygen.py -f py -o test.py auto_proxy --host ONION_URL_HERE -W SOCKS5:127.0.0.1:9050 -D -t ssl`

Everything is installed correctly, without errors whatsoever. But when I try to execute my python payload on my windows machine I get the following error. This is what I'm trying to debug but it's encoded.

The same happens with **pyinst** payloads.

![pupyerror](https://user-images.githubusercontent.com/61113942/106638051-18277d00-658c-11eb-85f4-5f648d35970b.png)

**Comment by analyserdmz** [comment]
Nevermind. I was trying to execute the payload with python version 3. Now it's working (running) but I don't get the shell, so, I still need a way to see the payload un-encoded, please.

**Comment by alxchk** [comment]
python 3 version is not released (yet)

**Comment by analyserdmz** [comment]
@alxchk yup this was my mistake. But still, how can I generate a payload that's not encoded?

**Comment by alxchk** [comment]
AFAIR --debug will not compile to *.pyc. Still, payload itself have a lot of packed parts. It's not to hide something, it's just works this way. If you want to do some research, you may just execute "pupy/pupy" as a module (it have __main__.py)
